pkg/simple/client/es/versions/error.go

```diff
@@ -0,0 +1,32 @@
+package versions
+
+type Error struct {
+	Status  int           `json:"status"`
+	Details *ErrorDetails `json:"error,omitempty"`
+}
+
+type ErrorDetails struct {
```

Comment:
Useless fields can be deleted.

Reply:
fixed, I have removed some fields by referring to the structure of other `error` packages.